awx/main/expect/isolated_manager.py

```diff
@@ -129,12 +134,12 @@ def dispatch(self, playbook):
         output = buff.getvalue()
         playbook_logger.info('Isolated job {} dispatch:\n{}'.format(self.instance.id, output))
         if status != 'successful':
-            event_data = {
-                'event': 'verbose',
-                'stdout': output
-            }
-            event_data.setdefault(self.event_data_key, self.instance.id)
-            CallbackQueueDispatcher().dispatch(event_data)
+            for event_data in [
```

Comment:
Sending this EOF event was necessary to make the UI happy when playbooks fail to launch on isolated nodes.